### PR TITLE
Dalai2547/sucu 33 component navbar

### PR DESCRIPTION
**Pull request by D33102**
nav bar

### CODE REVIEW

**Comment by macgeargear** [comment]
Could you please add your component in `playground.svelte` dai mai krub donggy

**Comment by D33102** [comment]
> Could you please add your component in `playground.svelte` dai mai krub donggy

**Comment by D33102** [comment]
> Could you please add your component in `playground.svelte` dai mai krub donggy

Dai ngubb

**Comment by macgeargear** [comment]
lgtm

**Review by macgeargear — APPROVED** [review]

File: static/sucu_logo_1.svg
Comment (punchanabu) [inline static/sucu_logo_1.svg:1]: I think putting this in `src/lib/icons` instead
https://stackoverflow.com/questions/71789244/where-to-put-images-with-sveltekit#:~:text=I%20recommend%20putting%20images%20under,a%20hash%20to%20the%20filename.

Comment (punchanabu) [reply static/sucu_logo_1.svg:1]: please check responsiveness na
image

Comment (punchanabu) [reply static/sucu_logo_1.svg:1]: image

**Pull request merged**